src/app/modules/group/http-services/get-group-members.service.ts

```diff
@@ -0,0 +1,64 @@
+import { HttpClient, HttpParams } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { appConfig } from 'src/app/shared/helpers/config';
+import { map } from 'rxjs/operators';
+
+interface RawMember {
+  id: string,
+  member_since: string|null;
+  action: 'invitation_accepted'|'join_request_accepted'|'joined_by_code'|'added_directly';
+  user: {
```

Comment:
Actually the `user` attribute is nullable. For now, you can filter out (in the service directly) the entries with null `user` as they are for members which are not users. (I thought this service was for user only but that's not the case)

Reply:
fixed